packages/klevu-ui/src/components/klevu-infinite-scroll/klevu-infinite-scroll.stories.ts

```diff
@@ -0,0 +1,113 @@
+import { MDXAutoFillMeta } from "../../storybookUtils"
+import { html } from "lit-html"
+import type { Meta, StoryObj } from "@storybook/web-components"
+import { KlevuInfiniteScroll } from "./klevu-infinite-scroll"
+import { ifDefined } from "lit-html/directives/if-defined.js"
+export const { argTypes, parameters, description, decorators } = MDXAutoFillMeta("klevu-infinite-scroll")
+
+const meta: Meta = {
+  title: "Atoms/Infinite Scroll",
```

Comment:
Should be in `Utils/Infinite Scroll` instead of Atoms. Atoms are simple UI components that are used in almost everywhere.

Reply:
renamed